### PR TITLE
ISPN-5446 8.0 Version Bump

### PR DESCRIPTION
**Pull request by tristantarrant**
https://issues.jboss.org/browse/ISPN-5446

### CODE REVIEW

**Comment by pruivo** [comment]
a bunch of failures in query module with 
```
org.infinispan.commons.CacheConfigurationException: ISPN000327: Cannot find a parser for element 'infinispan' in namespace 'urn:infinispan:config:7.2'. Check that your configuration is up-to date for this version of Infinispan.
```
ps. I didn't check the changes

**Comment by pruivo** [comment]
same error in cdi module

File: commons/.gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+/target/
+/target/
```
Comment (wburns) [inline commons/.gitignore]: Duplicate line

**Comment by wburns** [comment]
I wonder we break out all the parsers by independent version.  Would it be easier to just keep the extra parsers (jdbc cache, jpa etc.) as Parser72 and just have it handle 70,71,72,80 and if we add anything in 80 we can break it out?  The main config should be broken out to 80 as that for sure will be changed.  Or do we always want to break these out each major/minor version?

File: server/integration/endpoint/src/main/java/org/infinispan/server/endpoint/subsystem/EndpointSubsystemReader_7_2.java [hunk 2]
```diff
@@ -44,6 +44,7 @@
  *
  */
 class EndpointSubsystemReader_7_2 implements XMLStreamConstants, XMLElementReader<List<ModelNode>> {
+   public static final XMLElementReader<List<ModelNode>> INSTANCE = new EndpointSubsystemReader_7_2();
```
Comment (wburns) [inline server/integration/endpoint/src/main/java/org/infinispan/server/endpoint/subsystem/EndpointSubsystemReader_7_2.java]: I noticed these new INSTANCE fields all over?  What are they used for?  I don't see any references.

**Comment by tristantarrant** [comment]
I want to keep one parser per major version. Doing a Parser70 Parser71 and Parser72 was silly and pointless, since the changes are going to be additive-only.

File: server/integration/endpoint/src/main/java/org/infinispan/server/endpoint/subsystem/EndpointSubsystemReader_7_2.java (hunk 2 above)
Comment (tristantarrant) [inline server/integration/endpoint/src/main/java/org/infinispan/server/endpoint/subsystem/EndpointSubsystemReader_7_2.java]: They are referenced in Namespace.java

**Comment by tristantarrant** [comment]
Removed the .gitignore file from commons entirely. CI needs to upgrade to JDK 8 for this

File: server/integration/endpoint/src/main/java/org/infinispan/server/endpoint/subsystem/EndpointSubsystemReader_7_2.java (hunk 2 above)
Comment (wburns) [inline server/integration/endpoint/src/main/java/org/infinispan/server/endpoint/subsystem/EndpointSubsystemReader_7_2.java]: Ah yes I see now, thanks.

**Comment by galderz** [comment]
Let the fun begin!! :grinning:

**Comment by galderz** [comment]
Integrated

**Pull request closed without merging**